### PR TITLE
Adding the second part of Chapter 4 (regression) for cohort #1

### PR DESCRIPTION
**Pull request by A-Farina**
This PR provides the slides/notes from Cohort 1's discussion for chapter 4 part 2 (12 July 2021).

### CODE REVIEW

File: 04.Rmd [hunk 1]
```diff
@@ -10,18 +10,267 @@
...
+
+```{r}
+neighborhood_groups <- dat %>% 
+  dplyr::mutate(resid = residuals(house_lm)) %>% 
+  dplyr::group_by(Neighborhood) %>% 
+  dplyr::summarize(med_resid = median(resid),
+            cnt = n()) %>% 
+  dplyr::arrange(med_resid) %>% 
+  dplyr::mutate(cum_cnt = cumsum(cnt),
+        neighborhoodgroup = as_factor(ntile(cum_cnt, 5)))
```
Comment (jonthegeek) [inline 04.Rmd:140]: This line fails to render. It needs the `forcats::` prefix.
(there might be other errors after this one, but this kills the build)

**Review by jonthegeek — CHANGES_REQUESTED** [review]
I recommend running a Build on the book to make sure it renders properly!

File: 04.Rmd (hunk 1 above)
Comment (A-Farina) [reply 04.Rmd]: Thank you, I made that adjustment and built the book. It should work correctly now

**Review by jonthegeek — APPROVED** [review]
Looks good! Github Actions is making sure the book builds, if it does I'll merge this!

**Pull request merged**